community/STEERING-COMMITTEE.md

```diff
@@ -1,61 +1,122 @@
 # Knative Steering Committee
 
-The Knative Steering Committee (SC) defines, evolves, and defends the vision,
-values, mission, and scope of the project. _The Steering Committee is a
-work-in-progress._
+The Knative Steering Committee (KSC) is the ultimate authority for the Knative
+project, and controls all aspects of the project or delegates its authority
+to other committees.
+
+KSC has delegated technical guidance for the project to the
+[Technical Oversight Committee](TECH-OVERSIGHT-COMMITTEE.md). However, the KSC
+acts as the final escalation point for any and all decisions within the project.
 
 - [Charter](#charter)
+- [Committee Meetings](#committee-meetings)
 - [Committee Mechanics](#committee-mechanics)
 - [Committee Members](#committee-members)
+  - [Allocation of seats](#allocation-of-seats)
+- [Frequently asked questions](#frequently-asked-questions)
+- [Getting in touch](#getting-in-touch)
 
 ## Charter
 
-- Non-technical project oversight
+1. Define, evolve, and defend the vision, values, mission, and scope of the
+   project - to establish and maintain the soul of Knative.
+1. Define and evolve project governance structures and policies, including
+   project roles and how collaborators become members, approvers, leads,
+   and/or administrators. This includes policy for the creation and
+   administration of community groups, [working groups](WORKING-GROUPS.md) and
+   committees.
+1. Approve membership to the Technical Oversight Committee.
+1. Steward, control access, delegate access, and establishes processes regarding,
+   all Knative project resources and has the final say in the disposition of
+   those resources.
+1. Manage the Knative brand and decide which things can be called "Knative" and
+   how that mark can be used in relation to other efforts or vendors.
+1. Act as the final escalation point for any disputes or escalations from the
+   Technical Oversight Committee.
 
-  - Define policy for the creation and administration of community groups,
-    including [Working Groups](WORKING-GROUPS.md) and Committees.
+## Committee Meetings
 
-  - Define and evolve project governance structures and policies, including
-    project role assignment and contributor promotion.
+The Knative Steering Committee meets bi-weekly or as-needed.
```

Comment:
nit: `bi-weekly` can mean both twice a week or every other week. It would be worth clarifying your intent here.

"twice weekly" or "fortnightly" can be used.

Reply:
I'll clarify it should be "every two weeks"